src/Error/Debug/ConsoleFormatter.php

```diff
@@ -66,7 +66,10 @@ public static function environmentMatches(): bool
         // Windows environment checks
         if (
             DIRECTORY_SEPARATOR === '\\' &&
-            (!(bool)env('ANSICON') || env('ConEmuANSI') !== 'ON')
+            strpos(strtolower(php_uname('v')), 'windows 10') === false &&
```

Comment:
Do we need this? If we are running on windows and in bash.exe can't we safely assume we are in a bash emulator?

Reply:
That's actually for two different scenarios, as of Windows 10 the console host supports colors out of the box, the bash check would be for earlier Windows versions.